src/main/modules/SnippetsManager.js

```diff
@@ -28,6 +28,17 @@ class SnippetsManager {
         this.keyboardHandler.start()
 
         fixPath()
+        this._addNewFields()
+    }
+
+    _addNewFields() {
```

Comment:
`_addNewFields` -> `_ensureAllFieldsArePresent`? "Add new fields" implies new fields are always added, where in fact the function is just ensuring everything is set properly.